src/main/java/com/faforever/client/legacy/map/MapVaultParser.java

```diff
@@ -12,4 +13,9 @@
    * stable and this code breaks whenever the returned HTML changes.
    */
   List<MapInfoBean> parseMapVault(int page, int maxEntries) throws IOException;
+
+  //FIXME change map to string
```

Comment:
Should it be `id` instead of `map` in this comment? Also, I'd rather call it `mapId` to make it clear what ID needs to be passed.